esphome/components/waveshare_epaper/waveshare_epaper.cpp

```diff
@@ -2635,6 +2635,209 @@ void WaveshareEPaper7P5InV2alt::dump_config() {
   LOG_UPDATE_INTERVAL(this);
 }
 
+/* 7.50inV2 with partial and fast refresh */
+bool WaveshareEPaper7P5InV2P::wait_until_idle_() {
+  if (this->busy_pin_ == nullptr) {
+    return true;
+  }
+
+  const uint32_t start = millis();
+  while (this->busy_pin_->digital_read()) {
+    this->command(0x71);
+    if (millis() - start > this->idle_timeout_()) {
+      ESP_LOGE(TAG, "Timeout while displaying image!");
+      return false;
+    }
+    App.feed_wdt();
+    delay(10);  // NOLINT
+  }
+  return true;
+}
+
+void WaveshareEPaper7P5InV2P::reset_() {
+  if (this->reset_pin_ != nullptr) {
+    this->reset_pin_->digital_write(true);
+    delay(20);  // NOLINT
+    this->reset_pin_->digital_write(false);
+    delay(2);  // NOLINT
+    this->reset_pin_->digital_write(true);
+    delay(20);  // NOLINT
```

Comment:
Please don't just sprinkler NOLINT everywhere.
```suggestion
    delay(20);
    this->reset_pin_->digital_write(false);
    delay(2);
    this->reset_pin_->digital_write(true);
    delay(20);
```